analysis/src/Protocol.ml

```diff
@@ -97,7 +97,7 @@ let stringifyCompletionItem c =
     | None -> null
     | Some doc -> stringifyMarkupContent doc)
 
-let stringifyHover s = Printf.sprintf {|{"contents": "%s"}|} (Json.escape s)
+let stringifyHover value = Printf.sprintf {|{"contents": %s}|} (stringifyMarkupContent {kind = "markdown"; value})
```

Comment:
Use [MarkupContent](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#markupContentInnerDefinition). [MarkedString](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#markedString) and [MarkedString[]](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#markedString) are depreacted.